editor/src/clj/editor/settings.clj

```diff
@@ -13,18 +13,16 @@
 ;; specific language governing permissions and limitations under the License.
 
 (ns editor.settings
-  (:require [clojure.string :as string]
-            [clojure.java.io :as io]
+  (:require [camel-snake-kebab :as camel]
+            [clojure.string :as string]
             [dynamo.graph :as g]
-            [util.murmur :as murmur]
-            [editor.graph-util :as gu]
             [editor.core :as core]
-            [editor.settings-core :as settings-core]
             [editor.defold-project :as project]
-            [camel-snake-kebab :as camel]
-            [editor.workspace :as workspace]
+            [editor.graph-util :as gu]
             [editor.resource :as resource]
-            [service.log :as log]))
+            [editor.resource-node :as resource-node]
+            [editor.settings-core :as settings-core]
+            [editor.workspace :as workspace]))
```

Comment:
Mostly sorting. I guess I added `[editor.resource-node :as resource-node]` to the list.